docs/source/models/intervention_models/mncnh_pregnancy/iv_iron_antenatal/iv_iron_mncnh.rst

```diff
@@ -89,28 +88,46 @@ Hemoglobin exposure
 
 .. todo::
 
-  Update IV iron effect size to be consistent with new data from Chris T.
-  Also, assume no individual level heterogeneity despite having some data on this. (We chose not to model this in order to simplify the data prep for this model)
+  - Decide how we want to model uncertainty in the effect size of IV iron on hemoglobin concentrations.
+  - Decide which REVAMP study we should use for Nigeria and Ethiopia - one study looks at second trimester and one looks at third trimester IV iron administration (the latter [Pasricha-et-al-2025]_ reported a smaller effect size of 17.5g/dL). Currently we are using the second trimester study.
+  - [Pasricha-et-al-2023]_ reports ferritin as an outcome of their RCT, but ferritin was not included as an eligibility criterion for receiving IV iron.
+    This contradicts our assumption that IV iron is only administered to those with low ferritin levels and we need to decide how we want to handle this discrepancy.
+  - Assume no individual level heterogeneity despite having some data on this. (We chose not to model this in order to simplify the data prep for this model)
 
 .. list-table:: Maternal hemoglobin effect size
   :header-rows: 1
 
   * - Population
+    - Location
     - Effect size
     - Parameter uncertainty
     - Stochastic uncertainty
     - Note
   * - Pregnant simulants who attend later pregnancy ANC with test hemoglobin levels less than 100 g/L and test low ferritin levels
+    - Nigeria and Ethiopia
+    - +20.2 g/L
     - 
     - 
+    - From the REVAMP study (second trimester) [Pasricha-et-al-2023]_
+  * - Pregnant simulants who attend later pregnancy ANC with test hemoglobin levels less than 100 g/L and test low ferritin levels
+    - Pakistan
+    - +26.3 g/L
     - 
     - 
+    - From the RAPID study [Derman-et-al-2025]_ 
+
+.. note::
+
+  Unlike the REVAMP study, the RAPID study did include ferritin levels (serum ferritin <30 ng/mL) as an eligibility criterion for receiving IV iron, which is consistent with our assumption that IV iron is only administered to those with low ferritin levels.
```

Comment:
Interesting! This makes me think that we should maybe use this trial for all of our locations. Basically the hypothesized difference in effect sizes by region is due to differential prevalence of non-iron-responsive anemias, but if we are applying the low ferritin eligibility criterion we will limit the impact of this. There is still the fraction of the population that has low hemoglobin due to a non-iron-responsive anemia (like sickle cell or something) AND happens to have secondary low ferritin due to dietary insufficiency, so it wouldn't be exactly perfect though. Does that REVAMP study report anything on effect modification by ferritin status?